### DISCUSSION TITLE
_tkinter.TclError: character U+1f7a8 is above the range (U+0000-U+FFFF) allowed by Tcl

### DISCUSSION THREAD

**Discussion by renanozelo** [israel-dryer/ttkbootstrap#334] [Q&A] [answered]
Hello,

I'm receiving the following error when I try to instance the object Tableview:

"_tkinter.TclError: character U+1f7a8 is above the range (U+0000-U+FFFF) allowed by Tcl"

I know this error is related to the special characterers present in this object, but I couldn't solve it without remove them from the Tableview code and I don't want to do that. As other people don't have the some problem, I think is something can be set in my Python or Tkinter installations.

Can you help me on that?

PS: I'm using Python 3.7.4 and ttkbootstrap 1.9.0

Thanks

**Comment by israel-dryer** [comment, marked as answer]
You may be able to fix this by using a more recent version of Python (3.8, 3.9+), which will give you bug fixes in tkinter which come from tcl/tk.

Unfortunately, not all of the modern character ranges are supported by tcl/tk and this is what you are running into here.

**Reply by renanozelo** [reply]
Thanks!
